_static/custom.css

```diff
@@ -0,0 +1,18 @@
+.rst-content li,
```

Comment:
Minor note: This custom style caught me by surprise when I saw the bullets for bulleted lists outdented from the paragraph, since none of our other tech notes are formatted that way, and it made me wonder if the document was broken.

Reply:
I was running the technote with the latest documenteer to see how things were and noticed that unordered lists were broken completely so this was a bit of a temporary save, but I could definitely spend another couple of minutes on this...